src/health.ts

```diff
@@ -48,8 +48,9 @@ export class Health {
                 return this.healthz(opts);
             }
             return false;
-        } catch (err: unknown) {
-            if (err instanceof Error && err.name === 'AbortError') {
+        } catch (err: any) {
+            console.log(err);
```

Comment:
Can you please remove this.

Reply:
So embarrassing to leave debug printfs in the code!

done.